packages/shared/src/components/PromotionalBanner.tsx

```diff
@@ -10,6 +10,10 @@ import ProgressiveEnhancementContext from '../contexts/ProgressiveEnhancementCon
 import usePersistentState from '../hooks/usePersistentState';
 
 export default function PromotionalBanner(): ReactElement {
+  // Disable this component in Jest environment
+  if (typeof jest !== 'undefined') {
```

Comment:
I think we can use the constant variable we have here for this case:

https://github.com/dailydotdev/apps/blob/f687a80cbd1492fe316558638a8638edefc8824c/packages/shared/src/lib/constants.ts#L16-L17

Reply:
Well spotted! Should have used that one! 👀